backend/core/serializers.py

```
@@ -1230,6 +1230,77 @@
+    folder = serializers.UUIDField(required=False)
+    audit_name = serializers.CharField()
+    framework = serializers.CharField()
+    create_risk_assessment = serializers.BooleanField()
+    risk_assessment_name = serializers.CharField(required=False)
+    risk_matrix = serializers.CharField(required=False)
+
+    def save(self, **kwargs):
+        return self.create(self.validated_data)
+
+    def create(self, validated_data):
+        folder, _ = Folder.objects.get_or_create(
```

Ruff (0.8.2)

`Folder` may be undefined, or defined from star imports

(F405)